src/main/java/uk/gov/hmcts/reform/professionalapi/repository/OrganisationRepository.java

```diff
@@ -25,4 +27,10 @@
 
     @EntityGraph(value = "Organisation.alljoins")
     List<Organisation> findAll();
+
+    @Query(value = "select * from organisation as o, payment_account as p \n"
+            + "where p.organisation_id = o.id \n"
+            + "and p.pba_status = :pbaStatus \n"
+            + "order by p.created asc", nativeQuery = true)
+    List<Organisation> findByPbaStatus(@Param("pbaStatus")String pbaStatus);
```

Comment:
Any reason why native query is preferred over JPQL here?

Reply:
No specific reasons, any benefits of using JPQL instead?

Reply:
It is always recommended to use JPQL where possible. Native queries are specific to the database being used. In the future, if a decision was taken to move from postgres to a different database, say mysql or oracle, we have to modify all the native queries to the new database format. JPQL is database independent, the same query works for all the databases.

I don't want to block your PR because of this, but if it is a quick one, please change this to JPQL.

I think the below would work **if** there is a relationship defined already between organisation and payment_account tables:

```
select o from organisation o join fetch payment_account p \n"
            + "on p.organisation_id = o.id \n"
            + "where p.pbaStatus = :pbaStatus \n"
            + "order by p.created asc"
```

Reply:
thanks for the explanation @viswajit-kuna . I have modified the query, please review 6faeed5